apps/studio/components/interfaces/Organization/NewOrg/NewOrgForm.tsx

```diff
@@ -58,11 +58,11 @@ const NewOrgForm = ({ onPaymentMethodReset }: NewOrgFormProps) => {
   const stripe = useStripe()
   const elements = useElements()
 
-  const { plan } = useParams()
+  const { plan, name, kind, size, spend_cap } = useParams()
 
-  const [orgName, setOrgName] = useState('')
-  const [orgKind, setOrgKind] = useState(ORG_KIND_DEFAULT)
-  const [orgSize, setOrgSize] = useState(ORG_SIZE_DEFAULT)
```

Comment:
I only glanced at this, but if things are in params shouldn't we just use them, and not store them in useState as well ?